rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -90,66 +91,90 @@
 import org.mockito.Mockito;
 
 class BridgeSupportTest {
+    private static final String TO_ADDRESS = "0000000000000000000000000000000000000006";
+    private static final BigInteger DUST_AMOUNT = new BigInteger("1");
+    private static final BigInteger NONCE = new BigInteger("0");
+    private static final BigInteger GAS_PRICE = new BigInteger("100");
+    private static final BigInteger GAS_LIMIT = new BigInteger("1000");
+    private static final String DATA = "80af2871";
+    private static final Transaction tx = Transaction
+        .builder()
+        .nonce(NONCE)
+        .gasPrice(GAS_PRICE)
+        .gasLimit(GAS_LIMIT)
+        .destination(Hex.decode(TO_ADDRESS))
+        .data(Hex.decode(DATA))
+        .chainId(Constants.MAINNET_CHAIN_ID)
+        .value(DUST_AMOUNT)
+        .build();
+    private static final ECKey senderKey = RskTestUtils.getEcKeyFromSeed("sender");
+    private static final co.rsk.core.Coin LIMIT_MONETARY_BASE = new co.rsk.core.Coin(new BigInteger("21000000000000000000000000"));
+    private static final RskAddress contractAddress = PrecompiledContracts.BRIDGE_ADDR;
```

Comment:
```suggestion
    private static final RskAddress BRIDGE_CONTRACT_ADDRESS = PrecompiledContracts.BRIDGE_ADDR;
```

`contractAddress` sounds ambiguous to me, it could be any contract address. I prefer the one above or just `BRIDGE_ADDR`, if you go with the last one, there is no need to create a constant here; you just need to do a static import.

Reply:
i didn't change the name, just reorganized the lines